### DISCUSSION TITLE
Feature Request: support for groups

### DISCUSSION THREAD

**Discussion by daltonrooney** [trendyminds/isolate#32] [Ideas]
Hey there, I'm not sure how active development on this plugin is, but I wanted to throw out a few feature requests that I think would help with usability and I think could certainly be part of a premium edition of the plugin for users who need more features.

High on our list would be support for groups. Instead of isolating single users would be isolating an entire group of users at the same time.

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!
